portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,24 +34,46 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    static final String COMMENT_TABLE_NAME = "Comment";
+    static final String COMMENT_COLUMN_NAME = "comment";
+    static final String TIMESTAMP_COLUMN_NAME = "submit_time";
+    private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Convert comments to JSON
-    List<String> comments = new ArrayList<>();
-    comments.add("This is amazing!");
-    comments.add("10/10!");
-    comments.add("LGTM");
-    String json = convertToJsonUsingGson(comments);
+      Query query = new Query(COMMENT_TABLE_NAME).addSort(TIMESTAMP_COLUMN_NAME, SortDirection.DESCENDING);
+      PreparedQuery results = datastore.prepare(query);
+      List<String> commentsList = new ArrayList<String>();
+      List<Entity> limitedResults = results.asList(FetchOptions.Builder.withLimit(2));
```

Comment:
OOC why is this limited to two comments? Might be nice to use a constant for the 2 so future readers can understand why it's `2`.

Reply:
decided to hardcode the deletion of comments and how many comments are displayed -- ill make the variable names more clear